test/index.js

```diff
@@ -0,0 +1,4 @@
+import glob from 'glob';
+
+
+glob.sync('**/*-test.js', {realpath: true, cwd: __dirname}).forEach(require);
```

Comment:
It would be possible to avoid this file for tests, but since it is needed for coverage report, I use it for tests too.